tests/cbmc/proofs/s2n_stuffer_writev_bytes/s2n_stuffer_writev_bytes_harness.c

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+#include "api/s2n.h"
+#include "stuffer/s2n_stuffer.h"
+#include "utils/s2n_mem.h"
+
+#include <sys/param.h>
+#include <assert.h>
+
+#include <cbmc_proof/cbmc_utils.h>
+#include <cbmc_proof/make_common_datastructures.h>
+#include <cbmc_proof/proof_allocators.h>
+
+void s2n_stuffer_writev_bytes_harness() {
+    /* Non-deterministic inputs. */
+    struct s2n_stuffer *stuffer = cbmc_allocate_s2n_stuffer();
+    __CPROVER_assume(s2n_stuffer_is_valid(stuffer));
+    size_t iov_count;
+    __CPROVER_assume(iov_count < MAX_IOVEC_SIZE);
+    struct iovec iov[iov_count];
+    for(int i = 0; i < iov_count; i++) {
+        iov[i].iov_base = can_fail_malloc(iov[i].iov_len);
+    }
```

Comment:
Are we going to do this again? This should probably have an `allocate()` and `ensures()` function

Reply:
An allocated function would be more appropriate for reusability, but CBMC runtime goes over >30min if we do not use a static array here. I propose keep the code and open a GitHub issue to track this performance issue.

Reply:
Wow.  Yes, lets do that.

Reply:
https://github.com/awslabs/s2n/issues/2126 done.